### PR TITLE
Add action

### PR DESCRIPTION
**Pull request by mellowagain**
Fixes FP-1828

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    description: Title of the newly created event
+    required: true
+    default: GitHub Action
+  labels:
+    description: |
+      Labels to add to the event.
+      Format: key=value|key=value|key=value
+    required: true # For now these are required
+  time:
+    description: Time at which the event occurred. Defaults to current time
```
Comment (hatchan) [inline action.yml]: We should mention in which format the user should supply the time.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM debian:bullseye-slim
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends curl jq ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl https://fp.dev/fp/v0.8.0/x86_64-unknown-linux-gnu/fp -o /usr/bin/fp
```
Comment (hatchan) [inline Dockerfile]: nitpick, maybe we can move the version to a dockerfile ARG, that makes it easier to update down the line: https://docs.docker.com/engine/reference/builder/#arg

**Review by hatchan — APPROVED** [review]
Some minor nitpicks

File: Dockerfile (hunk 2 above)
Comment (mellowagain) [reply Dockerfile]: Oh that is very cool. The problem is we cannot pass build arguments to Docker as GitHub actions builds it automatically.

Comment (hatchan) [reply Dockerfile]: Yeah but we can specify a default value

**Pull request merged**